frontend/cypress/integration/common/health_indicators.ts

```diff
@@ -0,0 +1,62 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+
+Given('a healthy application in the cluster', function () {
+    this.targetNamespace = 'bookinfo';
+    this.targetApp = 'productpage';
```

Comment:
Just pointing out here that there's no guarantee that this app will be healthy when the test runs. It _should_ be but I've seen the various bookinfo apps go "unhealthy" for a bit just because of the nature of the environment. It'd be better to choose a healthy app from the list of applications rather than picking one ahead of time that may or may not be healthy. But I don't know how you choose a healthy app other than looking for the health icon which is what we're trying to test!

Reply:
> It'd be better to choose a healthy app from the list of applications rather than picking one ahead of time that may or may not be healthy. 

Uhm... I don't know. I would prefer this to be deterministic.
Choosing at runtime may make this harder to debug/fix if it breaks, no?

Although, honestly, the env itself is far from deterministic...